controllers/tasks.js

```diff
@@ -270,21 +270,27 @@ const updateTask = async (req, res) => {
     if (!task.taskData) {
       return res.boom.notFound("Task not found");
     }
-    if (req.body?.assignee) {
-      const user = await dataAccess.retrieveUsers({ username: req.body.assignee });
+    const requestData = { ...req.body };
+    if (requestData?.assignee) {
+      const user = await dataAccess.retrieveUsers({ username: requestData.assignee });
       if (!user.userExists) {
         return res.boom.notFound("User doesn't exist");
       }
+      if (!requestData?.startedOn) {
+        requestData.startedOn = new Date().getTime() / 1000;
```

Comment:
Does this necessary?
Why simply Date.now()

Reply:
I have taken the rounded-off value now.
Talking about divide by 1000 because, in some calculation on frontend, the value of it and the endsOn is used in that format so, for consistency.

Reply:
Date.now()/1000 is more preferable

Reply:
had to round it off so, endsOn and startedOn look coherent